other.yml

```diff
@@ -10,6 +10,7 @@ events:
       This format offers a place where SCS operators can meet and discuss operating SCS with the various topics around that.
       Jitsi server on https://conf.scs.koeln:8443/SCS-Tech
       Dial-In: +49-221-292772-611
+	  Coordinator: Felix Kronlage-Dammers <fkr[at]osb-alliance.com>
```

Comment:
I guess that this tab was not intended, nor?

Reply:
thanks @itrich for spotting that. fixed.